src/resources/swagger.yaml

```diff
@@ -261,7 +261,7 @@ paths:
           $ref: '#/components/responses/ForbiddenError'
         '500':
           $ref: '#/components/responses/ServerError'
-  /submission/program/{programId}/clinical/upload:
+  /submission/program/{programId}/clinical/:
```

Comment:
this is not related to the issue, but the word 'upload' here is not needed

Reply:
fyi, you'll have to update this in gateway too...

Reply:
yeah check here https://github.com/icgc-argo/platform-api/pull/98